llama_cpp/src/Utility/DataGenerator.cpp

```diff
@@ -0,0 +1,65 @@
+
+//
+//  Copyright © 2016 Project Llama. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#include "Utility/DataGenerator.h"
+
+#include <vector>
+#include <random>
+#include <string>
+
+template<>
+void llama::RandomData::generateRandomData<int>(std::vector<int> *targetVector, int count) {
+    std::random_device rd_int;
```

Comment:
Bad variable name. Should be intRandomDevice or randomDeviceInt or mRandomDeviceInt.